terraform/bambictf.tf

```diff
@@ -204,12 +204,12 @@ systemctl start wg-quick@internal
 TERRAFORMEOF
 }
 
-resource "hcloud_server" "moloch" {
+resource "hcloud_server" "elk" {
   name        = "moloch${count.index + 1}"

```

Comment:
will there ever be more than one elk? If not, we can drop the count (and it should be elk instead of moloch anyway)

Reply:
done